build.gradle

```
@@ -31,7 +31,7 @@ allprojects {
     // 2. In the GitHub UI, add a Release entry against the tag created in step 1. List any changes in release notes.
     // 3. Create a PR which bumps this from '0.10.0-SNAPSHOT' to '0.11.0-SNAPSHOT' in master to start the 0.11.0 track.
     // --
-    version = '0.56.0-SNAPSHOT'
+    version = '0.56.1-SNAPSHOT'
```

Should be the following if we're following the Wiki for [SDK Releases](https://wiki.mesosphere.com/display/ENG/%5BSDK%5D+Releasing+the+SDK).

```suggestion
    version = '0.57.0-SNAPSHOT'
```

---

Hm, that would currently break our CI, since' [were expecting](https://github.com/mesosphere/data-services-ci/blob/a85ff086c4665ca22b909e5ad24a833691ffc380/.teamcity/DataServices/buildTypes/ClusterConfig.kt#L160) a `mesosphere/dcos-commons:x.y.z` image to exist (where `x.y.z` is taken by ignoring an optional `-SNAPSHOT`). :-/